XROLE/storeManus
# Ch integrate travis #161382554

#### What does this PR do?
-Integrate Travis CI for continuous integration

#### Description of Task to be completed?
- Install Travis, Coveralls and nyc Packages
- Add and configure .travis.yml file
- Add  Coveralls and Coverage script in package.json 

#### How should this be manually tested?
- Navigate to https://github.com/XROLE/storeManus/compare/CH-integrate-travis-%23161382554?expand=1 
- Search for the build status
- The build status with a green check shows that Travis CI has been integrated successfully and the test is passing.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
- Story type - Chore
-Story title - Integrate Travis CI
-Story ID - #161382554
